lib/node_modules/@stdlib/array/base/cuany-by-right/docs/repl.txt

```diff
@@ -0,0 +1,52 @@
+cuanyByRight( x )
```

Comment:
Please see other packages. This file does not adhere to project conventions. See the REPL text guide in `/docs`.